src/main/java/com/example/store/controller/BookController.java

```diff
@@ -18,34 +21,46 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+@Tag(name = "Books management", description = "Endpoints to managing books")
 @RestController
 @RequestMapping(value = "/books")
 @RequiredArgsConstructor
 public class BookController {
     private final BookService bookService;
 
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    @Operation(summary = "Get all books",
+            description = "Get a page of all available books and can use sort")
+    public List<BookDto> getAll(Pageable pageable) {
+        return bookService.findAll(pageable);
     }
 
     @GetMapping("/{id}")
+    @Operation(summary = "Get book by id", description = "Get a available book by id")
     public BookDto findById(@PathVariable Long id) {
         return bookService.findById(id);
     }
 
     @GetMapping("/search")
-    public List<BookDto> search(@Valid BookSearchParametersDto bookSearchParametersDto) {
-        return bookService.search(bookSearchParametersDto);
+    @Operation(summary = "Get books by parametrs",
+            description = "Get a page of available books by parametrs and can use sort")
+    public List<BookDto> search(
+            Pageable pageable,
+            @Valid
+            BookSearchParametersDto bookSearchParametersDto
+    ) {
```

Comment:
such formatting is hard to read, please check other options. Probably
```suggestion
    public List<BookDto> search(Pageable pageable,
                                @Valid BookSearchParametersDto bookSearchParametersDto) {
```
or something like that